<launch>
	<node pkg="ros_color_detect" type = "ros_color_detect_node" name = "ros_color_detect_node" output="screen">
				
		<param name="img_width"                      type = "int"           value = "640"/>                 <!-- 이미지 크기 가로 -->
		<param name="img_height"                     type = "int"           value = "480"/>                 <!-- 이미지 크기 세로 -->        
			
	
		<param name="blue_b1"                        type = "int"         	value = "80"/>
		<param name="blue_g1"                        type = "int"          	value = "20"/>
		<param name="blue_r1"                        type = "int"          	value = "10"/>


		<param name="blue_b2"                        type = "int"         	 value = "225"/>
		<param name="blue_g2"                        type = "int"         	 value = " 105"/>
		<param name="blue_r2"                        type = "int"         	 value = " 110"/>

		
		
		
	
	</node>   
</launch>
